mappings/cvss_v3.json

```diff
@@ -727,11 +727,15 @@
           "children": [
             {
               "id": "privilege_escalation",
-              "cvss_v3": "AV:N/AC:H/PR:N/UI:R/S:U/C:N/I:L/A:N"
+              "cvss_v3": "AV:L/AC:H/PR:N/UI:N/S:U/C:L/I:L/A:N"
```

Comment:
Can we publish the CVSS mapping update within changelog?

Reply:
So we have yet to publish mapping changes in the `CHANGELOG` so we are going to continue to hold off for now.